src/OrbitGl/TimelineTicks.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TIMELINE_IMPL_H_
+#define ORBIT_GL_TIMELINE_IMPL_H_
+
+#include <cstdint>
+#include <set>
+#include <vector>
+
+namespace orbit_gl {
+
+constexpr uint64_t kSecondToNano = 1000 * 1000 * 1000;
+constexpr uint64_t kMinuteToNano = 60 * kSecondToNano;
+constexpr uint64_t kHourToNano = 60 * kMinuteToNano;
+
+class TimelineTicks {
```

Comment:
Add a class comment above the class and describe briefly what it is used for (https://google.github.io/styleguide/cppguide.html#Class_Comments)

Reply:
WDYT?